### ISSUE TITLE
Dereferencing is niet mogelijk voor gebruikte klassen en properties - open standaarden?

### ISSUE THREAD

**Issue by gezever** [Informatievlaanderen/OSLOthema-Waterkwaliteit#23]
Volgende url's, gebruikt in shacls, modellen en voorbeelddata (zie issue https://github.com/Informatievlaanderen/OSLOthema-Waterkwaliteit/issues/22) zijn niet dereferenceable.

https://purl.eu/ns/air-and-water/water-quality
http://def.isotc211.org/iso19103/2005/Names
http://def.isotc211.org/iso19103/2005/RecordsAndClassMetadata
http://def.isotc211.org/iso19103/2005/UnitsOfMeasure
http://def.isotc211.org/iso19108/2006/TemporalObjects
http://def.isotc211.org/iso19109/2005/GeneralFeatureModel
http://def.isotc211.org/iso19115/2006/DataQualityInformation
http://def.isotc211.org/iso19115/2006/MetadataEntitySetInformation
http://def.isotc211.org/iso19156/2011/GeneralFeatureInstance
http://def.isotc211.org/iso19156/2011/Measurement
http://def.isotc211.org/iso19156/2011/Observation
http://def.isotc211.org/iso19156/2011/SamplingFeature
http://def.isotc211.org/iso19156/2011/SpatialSamplingFeature
http://www.w3.org/ns/sosa/hasMember

**Comment by gezever** [comment]
online tool voor derefencing:
https://rdf-play.rubensworks.net

**Comment by gezever** [comment]
http://def.isotc211.org/iso19103/2005/Names
http://def.isotc211.org/iso19103/2005/RecordsAndClassMetadata
http://def.isotc211.org/iso19103/2005/UnitsOfMeasure
http://def.isotc211.org/iso19108/2006/TemporalObjects
http://def.isotc211.org/iso19109/2005/GeneralFeatureModel
http://def.isotc211.org/iso19115/2006/DataQualityInformation
http://def.isotc211.org/iso19115/2006/MetadataEntitySetInformation
http://def.isotc211.org/iso19156/2011/GeneralFeatureInstance
http://def.isotc211.org/iso19156/2011/Measurement
http://def.isotc211.org/iso19156/2011/Observation
http://def.isotc211.org/iso19156/2011/SamplingFeature
http://def.isotc211.org/iso19156/2011/SpatialSamplingFeature
is niet dereferenceable via content negotiation, wel via .rdf extensie

**Comment by GeertThijs** [comment]
> Volgende url's, gebruikt in shacls, modellen en voorbeelddata (zie issue #22) zijn niet dereferenceable.
> 
> https://purl.eu/ns/air-and-water/water-quality
> 

Achter deze zit hetzelfde probleem als in issue #24, de uri moet zijn [https://purl.eu/ns/air-and-water/water-quality]( https://purl.eu/ns/air-and-water/water).

**Comment by GeertThijs** [comment]
> http://def.isotc211.org/iso19103/2005/Names http://def.isotc211.org/iso19103/2005/RecordsAndClassMetadata http://def.isotc211.org/iso19103/2005/UnitsOfMeasure http://def.isotc211.org/iso19108/2006/TemporalObjects http://def.isotc211.org/iso19109/2005/GeneralFeatureModel http://def.isotc211.org/iso19115/2006/DataQualityInformation http://def.isotc211.org/iso19115/2006/MetadataEntitySetInformation http://def.isotc211.org/iso19156/2011/GeneralFeatureInstance http://def.isotc211.org/iso19156/2011/Measurement http://def.isotc211.org/iso19156/2011/Observation http://def.isotc211.org/iso19156/2011/SamplingFeature http://def.isotc211.org/iso19156/2011/SpatialSamplingFeature is niet dereferenceable via content negotiation, wel via .rdf extensie

Klopt. Prijs die we betalen om 1) zelf hiervoor geen uri's te moeten minten en 2) ganse ISO schema's te moeten reconstrueren ipv de delen die we nodig hebben.

**Comment by gezever** [comment]
> > Volgende url's, gebruikt in shacls, modellen en voorbeelddata (zie issue #22) zijn niet dereferenceable.
> > http://www.w3.org/ns/sosa/hasMember
> 
> De link was verkeerd, moet zijn https://www.w3.org/ns/sosa/hasMember. Is intussen verbeterd.

Dit zijn twee ideltieke url's
